src/main/java/lepimond/PeopleCLI.java

```diff
@@ -50,10 +67,73 @@ private void readNextCommand() throws Exception {
                     case "/select_all" -> command = new SelectAllCommand();
                     case "/select" -> command = new SelectCommand(scan.nextInt());
                     case "/help" -> command = new HelpCommand();
-                    default -> throw new SQLException();
+                    default -> throw new PeopleCLIException(I18n.getMessage("no_such_command"));
                 }
                 command.run();
             }
         }
     }
+
+    public static ResultSet executeQuery(String query) throws PeopleCLIException {
+        try {
+            return stmt.executeQuery(query);
```

Comment:
Close resources please.
```
    try (ResultSet resultSet = stmt.executeQuery(query)) {
        return resultSet;
    } catch (SQLException e) 
```

Reply:
This ResultSet is being closed later, after being returned to the method that called executeQuery()